source/index.html

```diff
@@ -4,9 +4,230 @@
+        </p>
+        <ul class="information__advantages information__advantages--gray advantages-list">
+          <li class="advantages-list__item">
+            <h3 class="advantages-list__title">
+              Поднимает настроение
+            </h3>
+            <p class="advantages-list__description">
+              Приложение позволит вам победить осеннюю хандру и депрессию буквально в несколько кликов!
+            </p>
+          </li>
+          <li class="advantages-list__item">
+            <h3 class="advantages-list__title">
+              Меняет мир вокруг
+            </h3>
+            <p class="advantages-list__description">
+              Сделайте снимок и украсьте его смайликом или текстовой подписью, чтобы усилить эффект
+            </p>
+          </li>
+          <li class="advantages-list__item">
+            <h3 class="advantages-list__title">
+              Заводит новых друзей
+            </h3>
+            <p class="advantages-list__description">
+              Ставьте лайки, комментируйте, делитесь фотографиями с друзьями и заводите новых
+            </p>
+          </li>
+        </ul>
+      </section>
+      <section class="reviews">
+        <h2 class="visually-hidden">Отзывы.</h2>
+        <ul class="reviews__list slider-list">
+          <li class="slider-list__item slider-list__item--current">
+            <blockquote class="blockquotes">
+              <cite class="blockquotes__author">
+                Николай Петров
+              </cite>
+              <p class="blockquotes__text">
+                «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+              </p>
+            </blockquote>
+          </li>
+          <li class="slider-list__item">
+            <blockquote class="blockquotes">
+              <cite class="blockquotes__author">
+                Николай Петров
+              </cite>
+              <p class="blockquotes__text">
+                «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+              </p>
+            </blockquote>
+          </li>
+          <li class="slider-list__item">
+            <blockquote class="blockquotes">
+              <cite class="blockquotes__author">
+                Николай Петров
+              </cite>
+              <p class="blockquotes__text">
+                «Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»
+              </p>
+            </blockquote>
+          </li>
+        </ul>
+        <div class="reviews__toggles slider-toggles">
+          <button class="slider-toggles__item slider-toggles__item--active" type="button" aria-label="Первый отзыв"></button>
```

Comment:
```suggestion
          <button class="slider-toggles__item slider-toggles__item--active" type="button" aria-label="Первый отзыв" aria-current="true"></button>
```

ты очень хорошо эти кнопки сделала - эти - это в смысле такие кружочки. Что тут, что в стоимости!